public_html/application/classes/Model/Group.php

```diff
@@ -58,17 +58,32 @@ function get_all_members ($type, $group_id)
 				// Prima di caricare le richieste pendenti, elimino quelle scadute
 				$ora = (time() - 86400);
 				ORM::factory('Group_character')
-					-> where( array( 'group_id' => $group_id, 'joined' => 0, 'date <' => $ora )) -> delete_all();
-				$members = ORM::factory('Group_character')->where( array( 'group_id' => $group_id) )->find_all();
+					->where('group_id', '=', $group_id)
+					->and_where('joined', '=', 0)
+					->and_where('date <', '<', $ora)
+					->delete_all();
+				$members = ORM::factory('Group_character')
+					->where( 'group_id', '=', $group_id)
+					->find_all();
 				break;
 			case "joined":
-				$members = ORM::factory('Group_character')->where( array( 'group_id' => $group_id, 'joined' => '1') )->find_all();
+				$members = ORM::factory('Group_character')
+					->where('group_id', '=', $group_id)
+					->and_where('joined', '=', '1')
+					->find_all();
 				break;
 			case "pendent":
 				// Prima di caricare le richieste pendenti, elimino quelle scadute
 				$ora = (time() - 86400);
-				ORM::factory('Group_character')->where( array( 'group_id' => $group_id, 'joined' => 0, 'date <' => $ora ))->delete_all();
-				$members = ORM::factory('Group_character')->where( array( 'group_id' => $group_id, 'joined' => 0) )->find_all();
+				ORM::factory('Group_character')
+					->where('group_id', '=', $group_id)
+					->and_where('joined', '=', 0)
+					->and_where('date <', '<', $ora)
```

Comment:
Got the operator right on this one, but we want ot remove it from the column string, the column is just `date`